### DISCUSSION TITLE
Learning rate during warmup steps

### DISCUSSION THREAD

**Discussion by qwertea** [kohya-ss/sd-scripts#426] [Q&A] [answered]
Is the learning rate during warmup steps 0? Or is it a % of the base LR? Or is it being gradually increased from 0 to the base LR? I understand all three are methods used in ML.

**Comment by laksjdjf** [comment, marked as answer]
its gradually increased from 0 to the base LR.
https://huggingface.co/docs/transformers/main_classes/optimizer_schedules#schedules
